ArduPlane/Parameters.cpp

```diff
@@ -210,7 +210,7 @@ const AP_Param::Info Plane::var_info[] = {
     // @Param: USE_REV_THRUST
     // @DisplayName: Bitmask for when to allow negative reverse thrust
     // @Description: This controls when to use reverse thrust. If set to zero then reverse thrust is never used. If set to a non-zero value then the bits correspond to flight stages where reverse thrust may be used. The most commonly used value for USE_REV_THRUST is 2, which means AUTO_LAND only. That enables reverse thrust in the landing stage of AUTO mode. Another common choice is 1, which means to use reverse thrust in all auto flight stages. Reverse thrust is always used in MANUAL mode if enabled with THR_MIN < 0. In non-autothrottle controlled modes, if reverse thrust is not used, then THR_MIN is effectively set to 0 for that mode.
-    // @Values: 0:MANUAL ONLY,1:AutoAlways,2:AutoLanding
+    // @Values: 0:MANUAL ONLY,1:AUTO_ALWAYS,2:AUTO_LAND,4:AUTO_LOITER_TO_ALT,8:AUTO_LOITER_ALL,16:AUTO_WAYPOINTS,32:LOITER,64:RTL,128:CIRCLE,256:CRUISE,512:FBWB,1024:GUIDED,2048:AUTO_LANDING_PATTERN,4096:FBWA,8192:ACRO,16384:STABILIZE,32768:THERMAL
```

Comment:
I'm not a fan of the change to all caps.. it is plane though so this is just a suggestion.